### PR TITLE
homework-functions

### PR DESCRIPTION
**Pull request by os-spirit**
- This PR contains homework on functions.
- All works are marked by number (1,2,3,4)
- Each task is contained in a separate file

### CODE REVIEW

File: js_basics/homework_functions/task1.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Завдання 1: Оголошення та виклик функції
+
+// Створіть функцію яка приймає два параметри: width і height.
+// Усередині функції обчисліть площу прямокутника, перемноживши width на height та поверніть результат з функції.
+// Викличте вашу функцію з аргументами (наприклад 5 і 10) і виведіть результат (площу прямокутника) в консоль.
+// Реалізуйте функцію трьома способами (function declaration, function expression, arrow function)
+
+
+// Declaration
+function rectangleArea(width, height) {
```
Comment (sttaran) [inline js_basics/homework_functions/task1.js]: function name should represent the action e.g. `calculateRectangleArea`, `getRectangleArea` etc

File: js_basics/homework_functions/task1.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    const rectangleAreaResult = width * height;
+    console.log(rectangleAreaResult);
```
Comment (sttaran) [inline js_basics/homework_functions/task1.js]: please return the value from the function

File: js_basics/homework_functions/task2.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Завдання 2: Засоби валідації
+
+// Створіть функцію яка приймає один параметр: age.
+// Усередині функції перевірте, чи age більше або рівне 18.
+// Якщо age відповідає умові, поверніть true, інакше поверніть false.
+// Викличте вашу функцію з аргументами 25 і 15, і виведіть результат (чи є особа дорослою) в консоль для кожного випадку.
+
+function isAdult (age) {
```
Comment (sttaran) [inline js_basics/homework_functions/task2.js]: you can make it easier 
```js
function isAdult (age) {
    return age >= 18
}
```

**Review by sttaran — CHANGES_REQUESTED** [review]

File: js_basics/homework_functions/task2.js (hunk 3 above)
Comment (os-spirit) [reply js_basics/homework_functions/task2.js]: Added minor improvements:
- Changed variable names in the first task
- Added return from a function when declaring it
- Console log is displayed outside the function (now it is when calling the function)
- Removed true / false keywords in the second task
[You can see these here](https://github.com/os-spirit/aqa-advanced-2/pull/4/commits/5ec85420b2b6b0829daeeca0951538f5b9e43c97)

**Review by sttaran — APPROVED** [review]

**Pull request merged**